store/tools/ics23-iavl/go.mod

```diff
@@ -0,0 +1,10 @@
+module github.com/confio/ics23-iavl
```

Comment:
need to update the module path, `..../cosmos-sdk/store/tools`.
Let's use one module for all these tools.